src/main/resources/META-INF/resources/primefaces/autocomplete/autocomplete.js

```diff
@@ -508,7 +515,7 @@ PrimeFaces.widget.AutoComplete = PrimeFaces.widget.BaseWidget.extend({
         var $this = this;
 
         //visuals and click handler for items
-        this.items.on('mouseover', function() {
+        this.items.off().on('mouseover.autocomplete', function() {
```

Comment:
is this correct to off() all? maybe just off('mouseover.autocomplete')?

Reply:
Fixed to the proper off `.off('click.autocomplete mousedown.autocomplete mouseover.autocomplete')`